server/modules/record/repository/validationReportRepository.js

```diff
@@ -45,7 +45,7 @@ const query = (surveyId) =>
       h.record_uuid = r.uuid 
       AND r.cycle = $/cycle/
       AND NOT r.preview
-    ORDER BY r.uuid, h.node_id`
+    ORDER BY h.node_id, r.uuid`
```

Comment:
why this change ?

Reply:
because we ordered the errors by uuid which is random and the node_id is number created in order

I can leave the query as it was and then I have to change the test to be "order" agnostic

what do you think? @minotogna

Reply:
@ramirobg94 let's use r.date_added, h.node_id